cloudigrade/config/settings/prod.py

```diff
@@ -9,23 +9,16 @@
 
 DJANGO_DEBUG = "False"
 
+# CLOUDIGRADE_ENVIRONMENT may have been loaded in base.py, but if not present, it would
+# have fallen back to generic defaults. However, in prod, we *require* that it be set
+# explicitly. By reloading here *without* a default, we halt app startup if not present.
 CLOUDIGRADE_ENVIRONMENT = env("CLOUDIGRADE_ENVIRONMENT")
 
+# TODO Why isn't HOUNDIGRADE_AWS_AVAILABILITY_ZONE configurable? Why is it special here?
```

Comment:
Because of this: https://github.com/cloudigrade/cloudigrade/blob/master/deployment/playbooks/roles/aws/defaults/main.yml#L40

But more realistically because long ago we agreed to setup and run everything from us-east-1a (since our 1a is not the same as everyone elses 1a), and as infra was manual setup, configs were not made to be easily changeable as we didn't expect a lot of change there.

We can go ahead and change this to pull from an env var, but then we should also update the playbook to source it from the same env var, and now the number of playbook inputs grows from 1 to 2 🙂 

Let me know what you think, I'm definitely open to making this better.

Reply:
Cloud-init is another spot where this will likely change.

Reply:
Good points! I'll add an inline comment summarizing your comment here, and I'll leave it to "future us" to refactor or remove later.